src/lib/menu/menu.spec.ts

```diff
@@ -485,6 +485,7 @@ describe('MdMenu', () => {
       fixture = TestBed.createComponent(SimpleMenu);
       fixture.detectChanges();
       fixture.componentInstance.trigger.openMenu();
+      fixture.componentInstance.closeCallback.calls.reset();
```

Comment:
Why is this necessary? The callbacks should be reset after each test, because the component is re-compiled.

Reply:
They were necessary when because Observable.of(null) was triggering a close event. Replaced with Observable.empty()

Reply:
`Observable.of(null)` will emit `null` immediately. `Observable.of()` won't.

Reply:
Are you saying that I should change it to Observable.of() instead of Observable.empty()? To me, Observable.empty() is more readable and makes the intent more clear.

But I don't mind change it to Observable.of() since the behavior would be the same as Observable.empty().

Reply:
Observable.empty() is definitely valid in this case. I just want to avoid bringing in an extra symbol from RxJS.

Reply:
Changed to Observable.of()